utils/rpms/packaging/rpm_chrootbuild

```diff
@@ -98,6 +102,10 @@ if [ -n "$DISTRO_VERSION" ]; then
     releasever_opt=("--config-opts=releasever=$DISTRO_VERSION")
 fi
 # shellcheck disable=SC2086
-eval mock --configdir "$mock_config_dir" -r "${CHROOT_NAME}"           \
-     ${repo_dels[*]} ${repo_adds[*]} --disablerepo=\*-debug*           \
-     "${releasever_opt[@]}" $MOCK_OPTIONS $RPM_BUILD_OPTIONS "$TARGET"
+if ! eval mock --configdir "$mock_config_dir" -r "${CHROOT_NAME}"           \
+          ${repo_dels[*]} ${repo_adds[*]} --disablerepo=\*-debug*           \
+          "${releasever_opt[@]}" $MOCK_OPTIONS $RPM_BUILD_OPTIONS "$TARGET"; then
```

Comment:
(lint) eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string). [SC2294]